src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/samples/Sample7_ConfiguringPipeline.cs

```diff
@@ -20,29 +20,26 @@ public partial class ConfigurationSamples
         public async Task ConfiguringPipeline()
         {
             // this instance will hold pipeline creation options
-            var options = ConfigurationClient.CreateDefaultPipelineOptions();
+            var options = new HttpPipelineOptions();
 
             // specify custon HttpClient
-            options.Transport = new HttpClientTransport(s_client);
-
-            // remove logging policy
-            options.LoggingPolicy = null; 
+            options.UseHttpClient(s_client);
```

Comment:
How will people know that transports are exclusive? i.e. will we also have UseXxx for all other transports?

Reply:
> i.e. will we also have UseXxx for all other transports?

Yes. But what other transports?